apps/music/js/metadata/core.js

```diff
@@ -57,11 +57,9 @@ var AudioMetadata = (function() {
     }
 
     // Read the start of the file, figure out what kind it is, and call
-    // the appropriate parser.  Start off with an 64kb chunk of data.
-    // If the metadata is in that initial chunk we won't have to read again.
+    // the appropriate parser.
     return new Promise(function(resolve, reject) {
-      var headersize = Math.min(64 * 1024, blob.size);
-      BlobView.get(blob, 0, headersize, function(header, error) {
+      BlobView.get(blob, 0, blob.size, function(header, error) {
```

Comment:
I am pretty sure that it doesn't. We already have a blob and AFAIK a slice is just a slice, not a copy of the buffer. I'll make sure, but there is no other clear solution to that. And it wasn't in the previous review because it is my test app that uncovered it: the unit test just take the whole blob.